README.md

```diff
@@ -13,106 +12,135 @@ Cert-issuer v3 is _not_ backwards compatible and does not support Blockcerts v2
 
 Ensure your docker image is running and bitcoind process is started
 
-1. Create an 'issuing address' and save the output as follows:
+1. Creating a wallet first with `bitcoin-cli createwallet “<wallet name>”`
 
-    ```
-    issuer=`bitcoin-cli getnewaddress`
-    sed -i.bak "s/<issuing-address>/$issuer/g" /etc/cert-issuer/conf.ini
-    bitcoin-cli dumpprivkey $issuer > /etc/cert-issuer/pk_issuer.txt
-    ```
+   ```
+   bitcoin-cli createwallet "testwallet"
+   ```
+
+2. Load wallet `bitcoin-cli loadwallet "<path to the directory of the created wallet>"`
+
+   ```
+   bitcoin-cli loadwallet "/root/.bitcoin/regtest/wallets/testwallet/"
+   ```
+
+   You can `bitcoin-cli listwallets` to check if your wallet is loaded
 
-2. Don't forget to save snapshots so you don't lose your work (see step 4 of client setup)
+3. Create an _issuing address_ and save the output as follows:
+
+   ```
+   issuer=`bitcoin-cli getnewaddress`
+   sed -i.bak "s/<issuing-address>/$issuer/g" /etc/cert-issuer/conf.ini
+   bitcoin-cli dumpprivkey $issuer > /etc/cert-issuer/pk_issuer.txt
+   ```
+
+   `sed` command allows us to quickly remove or replace the content without having to open a file.
+
+4. Don't forget to save snapshots so you don't lose your work (see step 4 of client setup)
 
 ## Issuing certificates
 
-1. Add your certificate to /etc/cert-issuer/data/unsigned_certificates/.
+1.  Add your certificate to `/etc/cert-issuer/data/unsigned_certificates/`:
 
     ```
     # To use a sample unsigned certificate as follows:
-    cp /cert-issuer/examples/data-testnet/unsigned_certificates/3bc1a96a-3501-46ed-8f75-49612bbac257.json /etc/cert-issuer/data/unsigned_certificates/
+    cp /cert-issuer/examples/data-testnet/unsigned_certificates/verifiable-credential.json /etc/cert-issuer/data/unsigned_certificates/
 
     # If you created your own unsigned certificate using cert-tools (assuming you placed it under data/unsigned_certificates):
     cp <cert-issuer-home>/data/unsigned_certificates/<your-cert-guid>.json /etc/cert-issuer/data/unsigned_certificates/
     ```
 
-2. Make sure you have enough BTC in your issuing address.
-
-    a. You're using bitcoind in regtest mode, so you can print money. This should give you 50 (fake) BTC:
+2.  Make sure you have enough BTC in your issuing address. You're using bitcoind in regtest mode, so you can print money. This should give you 50 (fake) BTC:
 
     ```
-    bitcoin-cli generate 101
+    bitcoin-cli -generate 101
     bitcoin-cli getbalance
     ```
 
-    b. Send the money to your issuing address -- note that bitcoin-cli's standard denomination is bitcoins not satoshis! (In our
-    app, the standard unit is satoshis.) This command sends 5 bitcoins to the address
+3.  Allow fallback fee in your `bitcoin.conf` (use Vim as the text editor). Add this line to `bitcoin.conf`: `fallbackfee=0.00001`
+
+    ```
+    vi /root/.bitcoin/bitcoin.conf
```

Comment:
can you specify the OS where you run this command? I suspect it might be different in MacOS and Windows.

Reply:
I suppose a quick note as to why this is important is great, since you spent time understanding it.

Reply:
I ran this command on Windows. I had to add that because I got this error:
```
 Fee estimation failed. Fallbackfee is disabled. Wait a few blocks or enable -fallbackfee.
```
But I think it's optional.